Virtual-Keyboard/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+.use-keyboard-input {
+  width: calc(100vw - 80px);
+  height: 300px;
+  padding: 10px;
+  position: relative;
+  top: 30px;
+  left: 30px;
+}
+.keyboard {
+  position: fixed;
+  left: 0;
+  bottom: -100%;
+  background: rgb(33, 24, 168);
+  width: 100%;
+  padding: 5px 0;
+  transition: bottom 1s;
+}
+.keyboard-hidden {
+  bottom: 0;
+}
+.keyboard__keys {
+  text-align: center;
+}
+.keyboard__key {
+  background: rgb(65, 54, 218);
+  width: 6%;
+  height: 45px;
+  border-radius: 5px;
+  vertical-align: top;
+  position: relative;
+  margin: 2.5px;
+  border: none;
+  outline: none;
+  color: white;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
+.keyboard__key:active {
+  background: rgb(50, 43, 151);
+}
+.key-active {
```

Comment:
![image](https://user-images.githubusercontent.com/11545672/110597774-0db36100-8192-11eb-98b1-c65a96d7a0b5.png)
It looks like as a modifaer.